README.md

```diff
@@ -14,35 +14,38 @@ by [OK Lab Leipzig](http://codefor.de/projekte/2014-07-01-le-lvz_polizeiticker_v
 Build and run the app with [npm](https://www.npmjs.com), [Grunt](http://gruntjs.com/) and [Gradle](https://gradle.org).
 The crawling and indexing of new articles is activated by the startup parameter --spring.profiles.active=crawl.
 
-### npm and Grunt
-
-Download client js dependencies with npm and package them with Grunt during postinstall phase.
-
-    npm install
-
-### Gradle
+### Gradle, npm and Grunt
 
 You can build an executable jar with gradle and run it as a separate process.
+The client js dependencies will be downloaded with npm and packaged with Grunt during the build phase from gradle.
 
-    ./gradlew build
-    java -jar build/libs/lvz-viz-*.jar
+```
```

Comment:
For codes I alway use this way. If it is not wanted, I can continue it as before.

Reply:
I think the use of backticks is also ok.